test/unit/utils/fs.spec.ts

```diff
@@ -109,3 +110,19 @@ describe('Test write method', () => {
 		await expect(write('', '')).rejects.toThrow();
 	});
 });
+
+describe('Test copyFile method', () => {
+	it('should copy file', async () => {
+		const srcPath = join(__dirname, '../../..', 'test/unit/fixtures/blockchain.db.tar.gz');
```

Comment:
Should we rewrite this to make it easier to read the path?
```suggestion
		const srcPath = join(__dirname, '../fixtures/blockchain.db.tar.gz');
```